apps/frontend/src/libs/hooks/use-query-parameters/use-query-parameters.hook.ts

```diff
@@ -1,16 +1,21 @@
 import { QueryParameterName } from "~/libs/enums/enums.js";
 import { useEffect, useSearchParams, useState } from "~/libs/hooks/hooks.js";
+import { type ValueOf } from "~/libs/types/types.js";
 
 type Parameters = {
 	isSavedToUrl?: boolean;
+	queryParameterName?: ValueOf<typeof QueryParameterName>;
 };
 
-const useSearch = ({ isSavedToUrl = true }: Parameters = {}): {
+const useQueryParameters = ({
```

Comment:
let's rename it because we have an option, not to save this to url, so previous name is confusing
```suggestion
const useSearchFilters = ({
```